### DISCUSSION TITLE
Tell us a 100Devs repo you would like spotlighted on the Marketplace!!!

### DISCUSSION THREAD

**Discussion by Michael-Futral** [CurrentlyUnderConstruction/.github#8] [Announcements]

**Comment by d-herz** [comment]
Hey Michael, I started putting together a repo with all the class-materials (and solutions, when provided) going back from the first class. My thinking was it would be an easy place for anyone in the catch-up crew to find the materials without having to go back and dig through the Discord 'follow-along materials' channel. 

Let me know what you think, I haven't finished yet but I am about halfway there:

https://github.com/d-herz/100Devs-Class-Materials

Also, where'd the readme go? Haha was going to update the schedule but it seems to have been taken down

**Reply by Michael-Futral** [reply]
That looks awesome!!! Think it would be an amazing addition to the marketplace! Currently working with Leon on better integrating this site into the 100Devs brand to better support it. It will be back up and functional, but we are working on integrating it into the 100 maintainers group for better control and support. So at the moment, the landing page is just "under construction" until we organize this site to meshed into the 100Devs brand. This Github organization will be back up and running soon, but just need some external adjustments to better support it. Sorry for the late response, yesterday and the holiday has been a bit crazy, and didn't have much access to a computer.

**Reply by d-herz** [reply]
Awesome sounds good! I'll continue building it up with the materials in the meantime then. And no worries! I figured you might be busy with holiday stuff.  But good to know, sounds like a good thing. We can talk more and put our thoughts together after the holiday stuff winds down 😌

**Reply by Michael-Futral** [reply]
Sorry for the delayed response. Yeah, will see what the 100 maintainers group meeting Leon has talked about goes this week. If this organization repo is something he doesn't want to include, still think it would be something useful to tweak the name and see if it is something the community would like. Still think a go to 100Devs location that people can easily find useful and active open-source repo projects would be great (great addition to the 'Together' app that is being created). May just need to tweak and adjust a few things to make it work.
